lib/smoke/verify-url.js

```diff
@@ -24,9 +25,16 @@ const verifyUrl = (testPage, browser, tests) => async () => {
 			return (typeof expectation === 'undefined' ? Promise.resolve() : Promise.resolve(testFn(testPage)).then(result => ({ name, results: [].concat(result) })));
 		});
 
-
 		const checkResults = await Promise.all(checks);
-		console.log(chalk`{bold Testing URL }({blue ${testPage.type}}): {underline.yellow ${testPage.url.toString()}}`);
+		//check for user option, use real url for logging
+		let realUrl = ''
+		if (testPage.requestHeaders['FT-Test-Host']) {
+			realUrl = new URL(`
+				${testPage.https ? 'https://' : 'http://'}
+				${testPage.requestHeaders['FT-Test-Host']}
+				${testPage.path || ''}
+			`)};
```

Comment:
the formatting here is a little confusing, the `}` is closing the `if` but here it looks like it's part of the `new URL`, and the semicolon isn't necessary after the `}`

```suggestion
			`);
		}
```